tests/integration/testdata/package/samconfig.toml

```diff
@@ -0,0 +1,10 @@
+version = 0.1
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "test-deploy-guided-set-parameter-0-aws-serverless-function-yaml-0-0-0"
```

Comment:
I don't think we should be committing this. The current tests should not require this. Committing this, runs the risk of other tests using these defaults and failing. 

For example, there can only be one S3 Bucket with a given name (names are global for S3). Hardcoding a bucket (event in a config file like this), will cause seemingly random failures for our package tests if someone other than you tries to run the tests.

Reply:
thats right. this file shouldn't be here.